src/devices/Ili9341/Ili9341.cs

```diff
@@ -0,0 +1,269 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Device.Gpio;
+using System.Device.Spi;
+using System.Drawing;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Iot.Device.Ili9341
+{
+    /// <summary>
+    /// The ILI9341 is a QVGA (Quarter VGA) driver integrated circuit that is used to control 240×320 VGA LCD screens.
+    /// </summary>
+    public partial class Ili9341 : IDisposable
+    {
+        private const int ScreenWidthPx = 240;
+        private const int ScreenHeightPx = 320;
+        private const int DefaultSPIBufferSize = 0x1000;
+        private const byte LcdPortraitConfig = 8 | 0x40;
+        private const byte LcdLandscapeConfig = 44;
+
+        private readonly int _dcPinId;
+        private readonly int _resetPinId;
+        private readonly int _backlitPin;
+        private readonly int _spiBufferSize;
+        private readonly bool _shouldDispose;
+
+        private SpiDevice _spiDevice;
+        private GpioController _gpioDevice;
+
```

Comment:
I think this goes toward issue #1644 that I logged. Considering this device is a fixed size, unless you have a generic interface to write against, why? The display device interface I modelled this from doesn't expose stuff like that either.

Reply:
The driver is fixed size. But as soon as you have a smaller screen attached, it can be map on any part of the memory. With this implementation, the developer will always have to take care of that. Otherwise, when writing the window, it's about setting the start point and end point for both the vertical and horizontal dimensions.
I'm fine as well leaving as it is. I'm working on similar driver (in native C/C++ with exposure in C"), for .NET nanoFramework and I have to take care of this. So I did implement that for all the drivers.